lms/static/sass/course.scss

```diff
@@ -1,15 +0,0 @@
-/*
```

Comment:
Oh, right! @singingwolfboy removed the need for ``.scss.mako`` files. Yeah, this makes sense now. I would just quickly check that no other compile files are importing this file.